Gruntfile.js

```diff
@@ -3,149 +3,43 @@
-					beautify : {
-						ascii_only : true,
-						quote_keys: true
-					}
-				},
-				files: [{
-					expand: true,
-					cwd: '<%= sourcePath %>/<%= assetDir %>/js',
-					src: '*.js',
-					dest: '<%= sourcePath %>/<%= assetDir %>/js/min'
-				}]
-			},
-			jsPlugins: {
-				options : {
-					beautify : {
-						ascii_only : true,
-						quote_keys: true
-					}
-				},
-				files: {
-					'<%= sourcePath %>/<%= assetDir %>/js/vendor/min/plugins.js': ['<%= sourcePath %>/<%= assetDir %>/js/vendor/**/*.js', '!<%= sourcePath %>/<%= assetDir %>/js/vendor/min/*.js'],
-				}
-			}
+	var path = require('path');
+
+	require('load-grunt-config')(grunt, {
+		// path to task.js files, defaults to grunt dir
+		configPath: path.join(process.cwd(), 'grunt'),
+
+		// auto grunt.initConfig
+		init: true,
+
+		// data passed into config.  Can use with <%= test %>
+		data: {
+			pkg: grunt.file.readJSON('package.json'),
+			banner: '/** \n' +
+				' * Automatically Generated - DO NOT EDIT \n' +
+				' * <%= pkg.name %> / v<%= pkg.version %> / <%= grunt.template.today("yyyy-mm-dd") %> \n' +
+				' */ \n\n',
+
+			sourcePath: 'src',
+			distPath: 'dist',
+			templateDir: 'templates',
+			assetDir: 'assets',
 		},
 
-		smushit: {
-			dist: {
-				src:  ['<%= sourcePath %>/<%= assetDir %>/img/**/*.png', '<%= sourcePath %>/<%= assetDir %>/img/**/*.jpg'],
-				dest: '<%= distPath %>/<%= assetDir %>/img'
-			}
+		// can optionally pass options to load-grunt-tasks.
+		// If you set to false, it will disable auto loading tasks.
+		loadGruntTasks: {
+			pattern: ['grunt-*', 'assemble'],
+			config: require('./package.json'),
+			scope: 'devDependencies'
 		},
 
-		usebanner: {
-			screenCSS: {
-				options: {
-					position: 'top',
-					banner: '<%= banner %>',
-					linebreak: true
-				},
-				files: {
-					src: [ '<%= distPath %>/<%= assetDir %>/css/*.css' ]
-				}
-			}
-		},
-
-		assemble: {
-		  options: {
-				assets: '<%= distPath %>/<%= assetDir %>',
-				layoutdir: '<%= sourcePath %>/<%= templateDir %>/layouts',
-				partials: ['<%= sourcePath %>/<%= templateDir %>/partials/**/*.hbs'],
-				flatten: true,
-		  },
-		  site: {
-				options: {
-					layout: 'default.hbs'
-				},
-				src: ['<%= sourcePath %>/<%= templateDir %>/*.hbs'],
-				dest: '<%= distPath %>'
-		  }
-		},
-
-		copy: {
-			assets: {
-				files: [{
-					expand: true,
-					cwd: '<%= sourcePath %>/<%= assetDir %>/',
-					src: ['**', '!scss/**'],
-					dest: '<%= distPath %>/<%= assetDir %>/'
-				}]
-			}
-		},
-
-		prettify: {
-			options: {
-				indent: 1,
-				indent_char: '  ',
-				brace_style: 'expand',
-				unformatted: ['a', 'code', 'pre']
-			},
-			dist: {
-				expand: true,
-				cwd: '<%= distPath %>',
-				ext: '.html',
-				src: ['*.html'],
-				dest: '<%= distPath %>'
-			}
-		}
+		//can post process config object before it gets passed to grunt
+		postProcess: function(config) {},
 
+		//allows to manipulate the config object before it gets merged with the data object
```

Comment:
Add space in between opening `//` and comment on this line and line 38 to match previous comments.